app/api/groups/get_granted_permissions.go

```diff
@@ -41,6 +41,8 @@ type grantedPermissionsViewResultRow struct {
 		// required: true
 		Title *string `json:"title"`
 		// required: true
+		Type *string `json:"type"`
```

Comment:
As `type` is a db enum, we specify the possible values.

see https://github.com/France-ioi/AlgoreaBackend/blob/master/app/api/items/get_children.go#L50